### ISSUE TITLE
ui: icon/button on dashboard project selector not

### ISSUE THREAD

**Issue by shwstppr** [apache/cloudstack#6667]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.17.0
~~~

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

N/A

##### SUMMARY

Icon/button in dashboard project selector is not working

On 4.16.1
[project-icon-working.webm](https://user-images.githubusercontent.com/153340/186141234-da3549dd-c456-424d-b4c9-f3ec7b833558.webm)

On 4.17.0
[project-icon-fail.webm](https://user-images.githubusercontent.com/153340/186141293-4b3c5755-291c-453f-a0da-c0d0c8f06156.webm)

##### STEPS TO REPRODUCE

~~~
Click on the icon/button on the left of the dashboard project selector
~~~

##### EXPECTED RESULTS

~~~
Animates, open project dropdown
~~~

##### ACTUAL RESULTS

~~~
Does nothing
~~~

**Comment by rohityadavcloud** [comment]
I think there was some discussions somewhere to remove the project icon.

**Comment by shwstppr** [comment]
It seems the behaviour changed due to vue3 changes. Would make sense to remove the custom suffix icon for the `select` icon.

**Comment by shwstppr** [comment]
Fixed with #6671
